src/commands/registries/azure/untagAzureImage.ts

```diff
@@ -6,25 +6,26 @@
 import { ProgressLocation, window } from "vscode";
 import { IActionContext } from "vscode-azureextensionui";
 import { ext } from "../../../extensionVariables";
-import { AzureTagTreeItem } from "../../../tree/registries/azure/AzureTagTreeItem";
+import { registryExpectedContextValues } from "../../../tree/registries/registryContextValues";
+import { RemoteTagTreeItem } from "../../../tree/registries/RemoteTagTreeItem";
 import { registryRequest } from "../../../utils/registryRequestUtils";
 
-export async function untagAzureImage(context: IActionContext, node?: AzureTagTreeItem): Promise<void> {
+export async function untagAzureImage(context: IActionContext, node?: RemoteTagTreeItem): Promise<void> {
     if (!node) {
-        node = await ext.registriesTree.showTreeItemPicker<AzureTagTreeItem>(AzureTagTreeItem.contextValue, { ...context, suppressCreatePick: true });
+        node = await ext.registriesTree.showTreeItemPicker<RemoteTagTreeItem>(registryExpectedContextValues.azure.tag, { ...context, suppressCreatePick: true });
     }
 
-    const confirmUntag: string = `Are you sure you want to untag "${node.fullTag}"? This does not delete the manifest referenced by the tag.`;
+    const confirmUntag: string = `Are you sure you want to untag "${node.repoNameAndTag}"? This does not delete the manifest referenced by the tag.`;
```

Comment:
Since this isn't destructive, do we need to prompt for it?  I realize that you aren't the one who made it a prompt, just wondering if we needed to have it.

Reply:
It is (partially) destructive